public/app/screens/main/navigation/style.scss

```diff
@@ -0,0 +1,12 @@
+@import "theme";
+
+.cc_navigation {
+  @include icon-bar();
+
+  .cc_navigation__link {
+    // .active is applied by foundation automatically
```

Comment:
Comment could be more descriptive:
`// .active is applied to selected link automatically by foundation`